sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -394,7 +395,7 @@ def _check_for_unsupported_fnapi_features(self, pipeline_proto):
       if (windowing_strategy.merge_status ==
           beam_runner_api_pb2.MergeStatus.NEEDS_MERGE and
           windowing_strategy.window_fn.urn not in (
-              common_urns.session_windows.urn, )):
+              common_urns.session_windows.urn, python_urns.PICKLED_WINDOWFN)):
         raise RuntimeError(
             'Unsupported merging windowing strategy: %s' %
             windowing_strategy.window_fn.urn)
```

Comment:
Don't we want to delete the clause below as well (and let the service reject it)?

Reply:
done.